hudi-common/src/main/java/org/apache/hudi/common/table/log/InstantRange.java

```diff
@@ -166,6 +169,28 @@ public boolean isInRange(String instant) {
     }
   }
 
+  /**
+   * Composition of multiple instant ranges in disjunctive form.
```

Comment:
Here by "disjuntive", do you mean the relation between these range is "OR", or do you mean these ranges are not consecutive? Meanwhile, in this function, we don't need to know their relation at all.

Reply:
Yeah, the `conjunctive` and `disjunctive` is terminology for relation algebra(SQL), `disjunctive` usually means `OR`.